profiles/car.lua

```diff
@@ -544,12 +544,13 @@ function way_function (way, result)
 end
 
 function turn_function (angle)
-  ---- compute turn penalty as angle^2, with a left/right bias
+  -- Use a sigmoid function to return a penalty that maxes out at turn_penalty
+  -- over the space of 0-180 degrees.  Values here were chosen by fitting
```

Comment:
We have two different notations here. OSRM internally operates on turn angles from `0` to `360` with `180` going straight.

Now in the profile, we use `-180` as `u-turn` to the left, `0` as going straight, `180` as u-turn to the right.